migrating_from_ocp_3_to_4/premigration-checklists.adoc

```diff
@@ -0,0 +1,90 @@
+[id="premigration-checks"]
+= Premigration checklists
+include::modules/common-attributes.adoc[]
+:context: premigration-checks
+
+toc::[]
+
+Before you migrate your application workloads with the {mtc-full} ({mtc-short}), review the following checklists.
+
+[id="source-cluster-checklist_{context}"]
+== Source cluster checklist
+
+* [ ] The cluster meets the link:https://docs.openshift.com/container-platform/3.11/install/prerequisites.html#hardware[minimum hardware requirements].
+* [ ] The {product-title} version is 3.7, 3.9, 3.10, 3.11.
+* [ ] The {mtc-short} version is the same on all clusters.
```

Comment:
Should this say that the version is the same as the target cluster?

Reply:
No. There is a use case of installing MTC on a remote cluster (neither target nor source), so there could be a third cluster involved.